queries/wof.jinja2

```diff
@@ -12,7 +12,7 @@ SELECT
       'area', area::bigint,
       'min_zoom', min_zoom,
       'max_zoom', max_zoom,
-      'is_landuse_aoi', CASE WHEN is_landuse_aoi = TRUE THEN TRUE END,
+      'is_landuse_aoi', is_landuse_aoi,
```

Comment:
Since 'min_zoom', 'max_zoom', 'is_landuse_aoi' and 'kind' all come from the WOF record, is it possible to do them all in YAML? Also `area` - all we're doing to it is converting the type, could that be done in a Python side processing function?